src/lib/tvgCanvasImpl.h

```diff
@@ -78,15 +78,11 @@ struct Canvas::Impl
 
         //Update single paint node
         if (paint) {
-            if (!paint->pImpl->update(*renderer, nullptr, compList, nullptr, RenderUpdateFlag::None)) {
-                return Result::InsufficientCondition;
```

Comment:
Is there any reason to delete the case where the update fails?
I think it's kind of maintaining compatibility.
I think it's better to keep passing edata as argument.

Reply:
There is no actual compatibility issues, still apis are changeable. 
Also, this return values are useless until now, users can't do anything about this.

Reply:
User can call update from (class paint)->push() and pass fail to user in case of task creation failure during update~> prepare process.
There is a problem with returning all cases to InsufficientCondition.
But I don't think there is any reason not to return an error.

Reply:
When it's meaningful then we can return values at that time.